src/components/Block/Block.tsx

```diff
@@ -0,0 +1,253 @@
+import React from 'react';
+// IMG
+import img from '../../img/BlockImg/img_1.svg';
+import homeIcon from '../../img/BlockImg/home_icon.svg';
+import searchIcon from '../../img/BlockImg/search_icon.svg';
+import cloudsIcon from '../../img/BlockImg/cloud.svg';
+import checkoutIcon from '../../img/BlockImg/checkout.svg';
+// STYLES
+import styled from 'styled-components';
+
+const BlockWrapper = styled.div`
+  cursor: default;
+  width: 330px;
+  height: 100%;
+  box-shadow: 5px 0 14px rgba(0, 0, 0, 0.1);
+  @media (max-width: 1280px) {
```

Comment:
Again – hardcoded sizes should be moved (just imagine if you'd like to change them and need to go to every file searching and repasting)